### ISSUE TITLE
Application onCreate Hook?

### ISSUE THREAD

**Issue by AnthonyUccello** [robolectric/robolectric#2456]
### Description
I am encountering this issue here:

https://github.com/robolectric/robolectric/issues/1981

TL:DR Cannot run the tests with parse

### Steps to Reproduce
Install parse and try to run a test.

### Robolectric & Android Version
Android version 21, and Robolectric version 3

I have created a work around in my Application file:

```
    @Override
    public void onCreate()
    {
        super.onCreate();
        //ParsePlugin.initializeParseWithApplication(this);
    }
```

But I'm curious if I can tell if Robolectric is running so I don't have to uncomment this to create a production apk. Kind of like you can check for CONFIG.DEBUG I'd like to check if Robolectric is running:

    @Override
    public void onCreate()
    {
        super.onCreate();
        if (!IS_ROBOLECTRIC_BUILD) ParsePlugin.initializeParseWithApplication(this);
    }

**Comment by jongerrish** [comment]
You shouldn't never have production code that tests if you are running in a unit testing environment, there is always a better way of doing this.

For the referenced issue, it is a parse initialization issue, someone familiar with the Parse library should take a look and determine why it is not reporting initialized.
